.github/workflows/tests.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         shell: bash
         run: |
           curl -fsS -o get-poetry.py https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py
-          python get-poetry.py --preview -y
+          python get-poetry.py -y
```

Comment:
You can port over to the new script now. https://github.com/python-poetry/poetry/blob/9e31f7a2f040f274148fec8fc6aae3721d2e6a90/.github/workflows/main.yml#L53-L66 `--preview` is no longer required i think.